miio/discovery.py

```diff
@@ -153,6 +154,7 @@
     "philips-light-sread1": PhilipsEyecare,  # name needs to be checked
     "philips-light-moonlight": PhilipsMoonlight,  # name needs to be checked
     "philips-light-rwread": PhilipsRwread,  # name needs to be checked
+    "huayi.light.pis123": Huizuo,
```

Comment:
This is the one used for mdns discovery and visible in `avahi-browse -a`, that's why it is here with `-`. Could you check if  the device is mdns discoverable?

Reply:
I just checked - devices are not discoverable  --> removed